bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/SeqConfigFlamingos2.java

```diff
@@ -29,7 +29,7 @@
             SPComponentType.ITERATOR_FLAMINGOS2;
 
     public static final ISPNodeInitializer<ISPSeqComponent, SeqConfigFlamingos2> NI =
-        new ComponentNodeInitializer<>(SP_TYPE, () -> new SeqConfigFlamingos2(), c -> new SeqConfigFlamingos2CB(c));
+        new ComponentNodeInitializer<>(SP_TYPE, SeqConfigFlamingos2::new, SeqConfigFlamingos2CB::new);
```

Comment:
Java has added some nice syntactic sugar and convenience classes since 1.7. It's still far too verbose and I prefer Kotlin any day, but at least it's less painful to use.